private-templates-service/client/src/App.tsx

```diff
@@ -61,6 +63,9 @@ const mapDispatchToProps = (dispatch: any) => {
     },
     userLogout: () => {
       dispatch(logout());
+    },
+    getConfig: () => {
```

Comment:
Once this function is called, I would expect the metric initialization stuff to be called because it needs the instrumentation key that will be provided by the backend. I know the metric code is currently in index.tsx. Does it need to be moved into app.tsx?